src/Codeception/Lib/InnerBrowser.php

```diff
@@ -1877,6 +1877,10 @@ protected function setCookiesFromOptions()
     protected function getNormalizedResponseContent()
     {
         $content = $this->_getResponseContent();
+        // Since strip_tags has problems with JS code that contains
+        // <= operators, these operators are replaced manually
+        // first to make sure strip_tags works as expected.
+        $content = preg_replace("/(<script.*>.*)(<=)(.*<\/script>)/is", "$1==$3", $content);
```

Comment:
This pattern replaces only one occurrence  of `<=` between the first `<script>` and the last `</script> tags.
Even if `.*` was changed to `.*?` it would replace only one occurrence  of `<=` in each script block.
What if there are more such conditions?

Reply:
That's true. And I just tested it with multiple `<script>` sections and it fails as well. I too think the first solution fixed the problem at hand, just not the edge cases.